.travis.yml

```diff
@@ -1,6 +1,7 @@
 dist: bionic
 
 language: python
+os: linux
```

Comment:
Please move this above line 1 because the `os` and the `dist` are directly related.

Reply:
it is easy / free to test on mac? would be nice to test at least one python version